cluster/manifests/postgres-operator/configmap.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: postgres-operator
+data:
+  service_account_name: operator
```

Comment:
We don't pre-define the `operator` account anywhere. So it has to be created here as well. Alternatively we can also use `kube-system` namespace and the `system` user.